### PR TITLE
[10.0] [MIG] account due list payment mode

### PR DESCRIPTION
**Pull request by Roodin**
It is a simpler migration that PR #130

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
Please update the README:

* [You can remove unneeded section](https://github.com/OCA/account-payment/pull/141/files#diff-fc9e5db76014967d607392a34c537bcbR34)

* [Update url runbot version to 10](https://github.com/OCA/account-payment/pull/141/files#diff-fc9e5db76014967d607392a34c537bcbR29)

**Comment by pedrobaeza** [comment]
@lreficent please give your last review after the changes

File: account_due_list_payment_mode/README.rst [hunk 1]
```diff
@@ -26,12 +26,11 @@ Usage
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/96/8.0
+   :target: https://runbot.odoo-community.org/runbot/96/10.0
 
 Known issues / Roadmap
```
Comment (LoisRForgeFlow) [inline account_due_list_payment_mode/README.rst]: I meant to remove completely the section.

**Review by LoisRForgeFlow — APPROVED** [review]
Just a clarification but I won't block - LGTM :+1:

**Comment by Roodin** [comment]
@lreficent You are very strict ;)

**Pull request merged**